samples/word/04-ranges/split-words-of-first-paragraph.yaml

```diff
@@ -1,6 +1,7 @@
-id: word-range-split-words-of-first-paragraph
-name: Range manipulation
-description: 'Show how to split a paragraph into word ranges and then traverses all the ranges to format each word, producing a "karaoke" effect.'
+order: 2
+id: word-ranges-split-words-of-first-paragraph
+name: Splitting a paragraph into ranges
```

Comment:
Consider changing "Splitting" to "Split" in snippet name (for consistency with how other snippet names start out)